src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.ts

```diff
@@ -34,14 +34,15 @@ export class RgwBucketListComponent {
   columns: CdTableColumn[] = [];
   buckets: object[] = [];
   selection: CdTableSelection = new CdTableSelection();
-
+  isTimeOut = true;
```

Comment:
This should start as `false` and I think `isStale `is a better var name.
```suggestion
  isStale= false;
```